packages/cozy-client/src/CozyClient.js

```diff
@@ -53,6 +52,7 @@ export default class CozyClient {
     return new QueryDefinition({ doctype, id })
   }
 
+  // eslint-disable-next-line no-unused-vars
```

Comment:
This disabling comment annoys me a bit, because I like that way of taking the fields that interest me and dismissing the rest using the spread operator: `{ _type, ...attributes }`. I suppose I could something like `{ _type, ..._ }` and remove the comment, but the code would loose a bit of readability. Do you see another way to fix the problem?

Reply:
For here I think we can remove it because you then do: `const document = { _type: type, ...attributes }` to replace `_type`.
For some other cases I think we could just take the parameters and then doing like here `{toReplace: replacement, ...attributes}` or use `Object.assign` to keep just properties we want in a new Object. I know it's not ideal but it's also more readable IMHO.